torch/onnx/utils.py

```diff
@@ -537,7 +537,7 @@ def _optimize_graph(
     operator_export_type: _C_onnx.OperatorExportTypes,
     _disable_torch_constant_prop: bool = False,
     fixed_batch_size: bool = False,
-    params_dict=None,
+    params_dict={},
```

Comment:
please consider
```python
if params_dict is None:
    params_dict = {}
```
instead of using mutable object as default value, https://google.github.io/styleguide/pyguide.html#2124-decision

Reply:
Thank you for your comment! I updated the code.